tests/openwisp2/sample_firmware_upgrader/tests.py

```diff
@@ -1,108 +1,55 @@
-import os
-from unittest import skipUnless
-
-from django.test import TestCase, TransactionTestCase
 from django.urls import reverse
```

Comment:
In the openwisp-ipam, I tried:
```
from openwisp_ipam.tests.test_admin import TestAdmin
from openwisp_ipam.tests.test_api import TestApi

# Monkey Patching to change the name of the app_label
# used tests. Change this value to your <app_label>.
TestAdmin.app_label = 'sample_ipam'
```

and it worked, isn't that simpler or Is there a reason you've avoided that here?

Reply:
Okay, adding tests would require monkey patching -- dirty solution, I am assuming that's why It's not used. :smile:

Reply:
I did not think about that. It It came natural to do it in the other way. I think both solutions are a bit hacky, it's just a matter of choice. Which one gives you less pain when you look at it? HEHE

Reply:
Fair enough! :smile:
I've also implement this one because it'll be easier for people not so well versed with python. :smile: